### PR TITLE
fix(DB/Creature): Patroling-Deathguards-chain-aggro-range

### PR DESCRIPTION
**Pull request by Stifler82**
## Changes Proposed:
- Fixed NPC: [Deathguard Abraham](https://wotlkdb.com/?npc=1735)
- Also fixed all [Brill](https://wowgaming.altervista.org/aowow/?zone=159) NPCs having the same problem.
- Now the Guards no longer interact with mobs, but only with "Players Opposite Faction" enemies.
**[EDIT]**
- I also added NPC: [Ironforge Mountaineer](https://wotlkdb.com/?npc=727) **(that this NPC also had the same problem).**

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4499
- Closes https://github.com/chromiecraft/chromiecraft/issues/59

## Tests Performed:
- Tested in game and working
- Tested on Windows 10 (64)
- **Video Test:** [Patroling Deathguards chain aggro range](https://youtu.be/GAgINSUQdac)

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Zoidwaffle** [comment]
I don't know these guards that well, but a lot of other guards, would also attack unfriendly NPC's and not just the opposing faction.

So I'm not sure this is Bliz-like tbh. Just a suggestion - and I suck at SAI so I could be wrong, but wouldn't some "Guard SmartAI" where they are faster at returning to their waypoints be better? Like aggro stuff really close, once OOC, get back to the WP again?

**Comment by Stifler82** [comment]
> I don't know these guards that well, but a lot of other guards, would also attack unfriendly NPC's and not just the opposing faction.
> 
> So I'm not sure this is Bliz-like tbh. Just a suggestion - and I suck at SAI so I could be wrong, but wouldn't some "Guard SmartAI" where they are faster at returning to their waypoints be better? Like aggro stuff really close, once OOC, get back to the WP again?

@Zoidwaffle 
But the problem wasn't the NPCs not returning to their waypoint, the problem was that the players used these NPCs to kill mobs and exping for free.

File: data/sql/updates/pending_db_world/rev_1614022899539097500.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 1652);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 2135);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 2136);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 2118);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 2134);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 1738);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 1742);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 10665);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 6 WHERE (`entry` = 10666);
+UPDATE `creature_template` SET `unit_flags` = 4608, `type` = 7 WHERE (`entry` = 727);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1614022899539097500.sql:29]: - https://www.azerothcore.org/wiki/Dealing-with-SQL-files read the Flags section
- you could group all of them like this
```SQL
UPDATE `creature_template` SET `unit_flags` = ...., `type` = 6 WHERE `entry` IN (1735,5725,3547,2314,1743,1515,2114,1518,10055,1745,2309,4773,4731,2311,2137,33996,1746,1652,2135,2136,2118,2134,1738,1742,10665,10666);
UPDATE `creature_template` SET `unit_flags` = ..., `type` = 7 WHERE (`entry` = 727);
```

**Review by Helias — CHANGES_REQUESTED** [review]

**Review by FrancescoBorzi — APPROVED** [review]
changed `entry = (727)` to `entry = 727` and approved

**Review by Helias — APPROVED** [review]

**Pull request merged**